recurring_contract/models/res_config_settings.py

```diff
@@ -0,0 +1,55 @@
+##############################################################################
+#
+#    Copyright (C) 2016-2022 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Emanuel Cino <ecino@compassion.ch>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+
+from odoo import models, fields, api
+
+from odoo.addons.recurring_contract.models.contract_group import ContractGroup
+
+
+class MandateStaffNotifSettings(models.TransientModel):
+    """ Settings configuration for any Notifications."""
+    _inherit = "res.config.settings"
+
+    # offset to know if we should generate current month or not
+    do_generate_curr_month = fields.Boolean(
+        string="Generate current month ?",
+        help="Define if the invoices should generate the current month by default or the next month. Ticked means we generate the current month and next month by default",
+        default=True,
+    )
+    # Day to know when we should stop generating invoices for a specific contract group on a contract creation
+    inv_block_day = fields.Selection(
+        selection="_day_selection",
+        string="Invoices Blocked Day",
+        help="Day at which we will suspend the invoices generation until the first of next month",
+        default="15",
+    )
+
+    def _day_selection(self):
+        return ContractGroup.day_selection()
+
+    @api.model
+    def get_values(self):
+        res = super().get_values()
+        res["inv_block_day"] = self.get_param_multi_company("compassion_nordic_accounting.inv_block_day")
+        res["do_generate_curr_month"] = self.get_param_multi_company("compassion_nordic_accounting.do_generate_curr_month")
+        return res
+
+    def set_values(self):
+        self._set_param_multi_company("compassion_nordic_accounting.invoice_block_day", self.inv_block_day)
+        self._set_param_multi_company("compassion_nordic_accounting.do_generate_curr_month", str(self.do_generate_curr_month))
+        super().set_values()
+
+    def get_param_multi_company(self, par_name):
+        return self.env["ir.config_parameter"].sudo().get_param(f"{par_name}_{self.env.company.id}", 0)
```

Comment:
The `get_param` method is not accessible from this module (see previous comment). This should be removed.